task/06-conditions-n-loops-tasks.js

```diff
@@ -192,7 +210,12 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+    let out = '';
+    out += (isStartIncluded)? '[' : '(' ;
+    if (a < b) out +=  a + ', ' + b ;  
```

Comment:
if you do an assignment it's better to move the body on the next line and use brackets